x/oracle/spec/02_state.md

```diff
@@ -0,0 +1,27 @@
+<!--
+order: 2
+-->
+
+# State
+
+The oracle module manages the address of the Oracle and the ICQ state.
+
+<!-- TOC 2 -->
+- [State](#state)
```

Comment:
```suggestion
```

The top-level header doesn't need to be in the TOC, and isn't included when you `make update-tocs`.